examples/custom-backend/main.rs

```diff
@@ -37,14 +37,14 @@ enum CustomCommands {
 fn run(ui: &mut Ui) -> Result<(), CommandError> {
     let app = CustomCommands::augment_subcommands(default_app());
     let (mut command_helper, matches) = parse_args(ui, app, std::env::args_os())?;
-    let mut backend_factories = BackendFactories::default();
+    let mut store_factories = StoreFactories::default();
```

Comment:
I think naming the 'type' files "type" would be best, and it would be good if the dir names matched the class names.  For Backend that's a bit of an ambiguous question because both the 'Store' and 'Backend' classes have a claim; I don't have a strong opinion there and will defer to you for any renames of those, I won't change those in my chain.